### PR TITLE
ADMINUI-2268 Support new account setting for Triton Container Naming …

### PR DESCRIPTION
**Pull request by Arc311**
…Service (CNS)

### CODE REVIEW

File: lib/users.js [hunk 1]
```diff
@@ -376,6 +376,11 @@ function update(req, res, next) {
         } else {
             changes['registered_developer'] = 'false';
         }
+        if (req.body.triton_cns_enabled) {
```
Comment (YZaicev) [inline lib/users.js]:
```
changes['triton_cns_enabled'] = req.body.triton_cns_enabled || 'false'
```

File: lib/users.js [hunk 2]
```diff
@@ -457,7 +462,8 @@ function createUser(req, res, next) {
         phone: params.phone,
         tenant: params.tenant || '',
         approved_for_provisioning: (params.approved_for_provisioning || 'false'),
```
Comment (YZaicev) [inline lib/users.js]: remove parentheses

File: lib/users.js [hunk 3]
```diff
@@ -457,7 +462,8 @@ function createUser(req, res, next) {
...
-        registered_developer: (params.registered_developer || 'false')
+        registered_developer: (params.registered_developer || 'false'),
```
Comment (YZaicev) [inline lib/users.js]: same

File: lib/users.js [hunk 4]
```diff
@@ -457,7 +462,8 @@ function createUser(req, res, next) {
...
+        triton_cns_enabled: (params.triton_cns_enabled || 'false')
```
Comment (YZaicev) [inline lib/users.js]: same

**Comment by YZaicev** [comment]
:1234:

**Comment by askfongjojo** [comment]
Merged as 3fb4a79

**Pull request closed without merging**